tests/ui_tests/app/app.py

```diff
@@ -41,6 +41,8 @@ def __init__(self, browser="Chrome"):
         self.opts.add_argument("--start-maximized")
         self.opts.add_argument("--enable-logging")
         self.opts.add_argument("--enable-automation")
+        if browser == "Chrome":
+            self.opts.add_argument("--window-size=1366,768")
```

Comment:
So why we have [`set_window_size`](https://github.com/arenadata/adcm/blob/423004f3ae58a8e6dc947b87e720b566989a20d7/tests/ui_tests/app/app.py#L72) below?

Reply:
@annademina
Do we need it now for some reason?
I didn't test it without `set_window_size` in non-headless mode, so not sure how does it affect debug

Reply:
as I remember we decided to set this window size to check if there are any problems with the website layout? Not only because of popups and etc.

Reply:
The thing is that we have 2 lines that setting window size:
- this one sets it via options (and works in headless mode)
- the one that @madlabman mentioned comes after this one and sets resolution via .set_window_size method that works in non-headless mode, but seems to have no effect on headless one, at least for Chrome

Maybe it works with Firefox (but as I know it's about 1366x768 by default in headless)

Reply:
We'll test it against Firefox and remove it if it's required no more.
Just not as part of current PR.

Reply:
I suggest to keep the wider one (by driver options).

Reply:
done.
added same for Firefox